git/git-cheatsheet-de.md

```diff
@@ -33,6 +33,14 @@ git config --global core.eol lf
 
 ## Branches
 
```

Comment:
Bitte in Markdown immer nur eine Leerzeile am Stück. Siehe dazu: http://www.cirosantilli.com/markdown-style-guide/#newlines